packages/cli/src/zosjobs/list/jobs/Jobs.definition.ts

```diff
@@ -39,6 +39,12 @@ export const JobsDefinition: ICommandDefinition = {
                 "You can specify a wildcard according to the z/OSMF Jobs REST endpoint documentation, " +
                 "which is usually in the form \"JOB*\".",
             type: "string"
+        },
+        {
+            name: "execdata", aliases: ["ed"],
+            description: "Use this option to retrieve execution data for jobs via the z/OSMF REST API.",
+            type: "bool",
```

Comment:
Type should be `"boolean"` based on the types included in the `CommandOptionType` enum:
https://github.com/zowe/imperative/blob/4926f56baf27fb1e36c797f26a7dcad79009bc83/packages/cmd/src/doc/option/ICommandOptionDefinition.ts#L26-L34